### PR TITLE
Bug 1162977 - Expand the raptor launch test suite on gaia

### PR DESCRIPTION
**Pull request by rwood-moz**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d9d2366fe509814821d178256de80d14cd4bda10

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=38c71dd157ff8142ecb9317544b863ea75044dc5

File: tests/taskcluster/bin/raptor_graph [hunk 1]
```diff
@@ -250,15 +259,15 @@ getEmulator()
...
+        var newSuiteName = oldSuiteName.substring(0, oldSuiteName.indexOf('test')) + "test (on base -1 commit)";
+        config[raptorTask].params.raptorSuiteName = newSuiteName;
         config[raptorTask].params.gaiaRev = gaiaBaseMinusOne;
       }
       if (process.env.GITHUB_PULL_REQUEST === '0' && config[raptorTask].build === "patch") {
-        var app = config[raptorTask].params.raptorApps;
-        suiteName = app[0].toUpperCase() + app.slice(1) + " app launch test (on base)";
-        config[raptorTask].params.raptorSuiteName = suiteName;
+        var oldSuiteName = config[raptorTask].params.raptorSuiteName;
+        var newSuiteName = oldSuiteName.substring(0, oldSuiteName.indexOf('test')) + "test (on base)";
```
Comment (gregarndt) [inline tests/taskcluster/bin/raptor_graph]: I think the only caveat here is that the TH name must never contain  the word "test" in it.

Comment (rwood-moz) [inline tests/taskcluster/bin/raptor_graph]: Right, good point thanks

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1bcb69952e258b5061c182f8f50e1b7acfe38916

**Pull request merged**